_inc/lib/class.color.php

```diff
@@ -419,7 +419,7 @@ public function toXyz()
 		}
 		$rgb = $rgb_new;
 
-		//Observer. = 2�, Illuminant = D65
+		//Observer. = 2�, Illuminant = D65
```

Comment:
It doesn't affect anything, but this character shouldn't be like this. It's also like this in the r128444-wpcom.